galsim/galaxy_sample.py

```diff
@@ -31,39 +31,23 @@
 # Note: This isn't used anywhere.  This is just informational, really.
 cosmos_pix_scale = 0.03
 
-class COSMOSCatalog:
+class GalaxySample:
     """
-    A class representing a random subsample of galaxies from the COSMOS sample with F814W<25.2
-    (default), or alternatively the entire sample with F814W<23.5.
+    A class representing a random subsample of galaxies from an arbitrary deep survey.
 
     Depending on the keyword arguments, particularly ``use_real``, the catalog will either have
-    information about real galaxies, and/or parametric ones.  To use this with either type of
-    galaxies, you need to get the COSMOS datasets in the format that GalSim recognizes; see
+    information about real galaxies, and/or parametric ones.
```

Comment:
I realize this isn't new text.  But I think it would be more accurate to say `use_real` tells the software what type of information you want to use for making the sims, right?  I mean, a catalog might have both and `use_real` chooses between them.

Reply:
Right.  This is talking about the sample, not the source file.  (Catalog is ambiguous here.)  I changed it to "Depending on the keyword arguments, particularly ``use_real``, a `GalaxySample` may be able to generate real galaxies, parametric galaxies, or both."